### PR TITLE
Feature/recipe styles

### PR DESCRIPTION
**Pull request by martynaspuskunigis**

### CODE REVIEW

File: src/app/components/active-recipe/active-recipe-view.scss [hunk 1]
```diff
@@ -1,51 +1,97 @@
...
+            padding-bottom: 12px;
+            font-weight: bold;
         }
 
-        .active-recipe-title {
-            text-transform: uppercase;
-            letter-spacing: 2px;
+        img {
+            width: 620px;
+            height: 300px;
```
Comment (justinasalt) [inline src/app/components/active-recipe/active-recipe-view.scss]: is this responsive?

File: src/app/components/recipe/recipes-item-view.scss [hunk 2]
```diff
@@ -5,23 +5,50 @@
     box-shadow: -1px 2px 34px -13px rgba(0, 0, 0, 0.75);
     display: flex;
     flex-direction: column;
-    width: 310px;
+    width: 360px;
     padding: 15px 30px;
-    margin-bottom: 30px;
+    margin: 30px;
```
Comment (justinasalt) [inline src/app/components/recipe/recipes-item-view.scss]: why no padding?

File: src/app/components/active-recipe/active-recipe-view.scss (hunk 1 above)
Comment (martynaspuskunigis) [reply src/app/components/active-recipe/active-recipe-view.scss]: Yes it is

File: src/app/components/recipe/recipes-item-view.scss [hunk 3]
```diff
@@ -5,23 +5,50 @@
...
             white-space: nowrap;
             overflow: hidden;
             text-overflow: ellipsis;
+            padding-top: 10px;
+
+            .recipes-title {
+                color: #e74c3c;
+                text-transform: uppercase;
+                font-size: 1.1rem;
+                padding: 1.8rem 0 0.2rem 0;
```
Comment (DonatasBrazionis) [inline src/app/components/recipe/recipes-item-view.scss]: Why you sometimes use `rem`, sometimes `px`?

File: src/app/components/recipe/recipes-item-view.scss (hunk 2 above)
Comment (martynaspuskunigis) [reply src/app/components/recipe/recipes-item-view.scss]: Margin makes space after the border.

File: src/app/components/recipe/recipes-item-view.tsx [hunk 4]
```diff
@@ -41,28 +41,30 @@ export class RecipesItemView extends React.Component<Props> {
...
                 <div className="recipe-text">
-                    <a className="recipes-title" data-tip data-for={this.props.recipe.recipe_id}>
-                        {this.props.recipe.title}
-                    </a>
-                    <ReactTooltip id={`${this.props.recipe.recipe_id}`} type="dark" effect="float">
-                        <span>{this.props.recipe.title}</span>
-                    </ReactTooltip>
+                    <div className="recipe-header">
+                        <div
+                            className={classNames(
```
Comment (DonatasBrazionis) [inline src/app/components/recipe/recipes-item-view.tsx]: Maybe use `className={ this.props.isFavorite ? "fas fa-star star-icon-full" : "far fa-star star-icon-empty"}`?

**Pull request merged**